plugin/pkg/admission/initialresources/hawkular.go

```diff
@@ -95,11 +95,7 @@ func (hs *hawkularSource) GetUsagePercentile(kind api.ResourceName, perc int64,
 		m = append(m, metrics.Tenant(namespace))
 	}
 
-	p, err := metrics.ConvertToFloat64(perc)
```

Comment:
The same thing, just casting to `float64`: https://github.com/hawkular/hawkular-client-go/blob/master/metrics/helpers.go#L47-L48. Since `perc` is defined as a `int64` in the method signature, I thought we could remove the need for an external helper call and do the same thing natively.